rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -98,12 +101,13 @@ public class PeerExplorer {
     public PeerExplorer(List<String> initialBootNodes,
                         Node localNode, NodeDistanceTable distanceTable, ECKey key,
                         long reqTimeOut, long updatePeriod, long cleanPeriod, Integer networkId,
-                        PeerScoringManager peerScoringManager, boolean allowMultipleConnectionsPerHostPort) {
+                        PeerScoringManager peerScoringManager, boolean allowMultipleConnectionsPerHostPort, long maxBootRetries) {
         this.localNode = localNode;
         this.key = key;
         this.distanceTable = distanceTable;
         this.updateEntryLock = new ReentrantLock();
         this.networkId = networkId;
+        this.initialBootNodes = initialBootNodes;
```

Comment:
```suggestion
        this.initialBootNodes = Collections.unmodifiableList(new ArrayList<>(initialBootNodes));
```
minor thing, just to be on a safer side

Reply:
just added all the changes, running some tests to check everything it's ok